src/components/layout/NavBar.jsx

```diff
@@ -74,83 +65,51 @@ class NavBar extends Component {
 
   render() {
     const navItems = this.props.navItems.map(
-      (item, index) => {
+      (item) => {
         const navButton = (<div
           key={item.link}
           ref={this.getNavButtonRef(item.link)}
           className='nav-bar__link nav-bar__link--right'
           onMouseOver={() => this.updateTooltip(item)}
           onMouseLeave={() => this.updateTooltip(null)}
         >
-          { item.link.startsWith('http') ?
-            (<a href={item.link}>
-              <NavButton
-                item={item}
-                dictIcons={this.props.dictIcons}
-                isActive={this.isActive(item.link)}
-                onActiveTab={() => this.props.onActiveTab(item.link)}
-                tabIndex={index + 1}
-              />
-            </a>)
-            :
-            (<Link to={item.link}>
-              <NavButton
-                item={item}
-                dictIcons={this.props.dictIcons}
-                isActive={this.isActive(item.link)}
-                onActiveTab={() => this.props.onActiveTab(item.link)}
-                tabIndex={index + 1}
-              />
-            </Link>)
-          }
+          <NavButton
+            item={item}
+            dictIcons={this.props.dictIcons}
+          />
         </div>);
         return this.canUserSeeComponent(item.name) ? navButton : null;
       });
 
     // added for backward compatibility
     // should always add homepageHref to components in portal config in the future
-    const homepageHref = components.homepageHref || config.homepageHref;
+    const homepageHref = components.homepageHref || config.homepageHref || '';
 
     return (
       <div className='nav-bar'>
         <header className='nav-bar__header'>
           <nav className='nav-bar__nav--info'>
             <div className='nav-bar__logo'>
-              {homepageHref ? (
-                <a href={homepageHref}>
-                  <img
-                    className='nav-bar__logo-img'
-                    src='/src/img/logo.png'
-                    alt={components.homepageAltText || 'Gen3 portal logo'}
-                  />
-                </a>
-              ) : (
-                <Link
-                  to=''
-                  onClick={() => this.props.onActiveTab('')}
-                  onKeyPress={() => this.props.onActiveTab('')}
-                >
-                  <img
-                    className='nav-bar__logo-img'
-                    src='/src/img/logo.png'
-                    alt={components.homepageAltText || 'Gen3 portal logo'}
-                  />
-                </Link>
-              )
-              }
+              <NavLink exact to={homepageHref}>
```

Comment:
would this work if the `homepageHref` is an external URL? I remember the `NavLink` can only route you on client side 🤔

Reply:
will look into what happens, good catch